pipeline/batch.go

```diff
@@ -79,8 +90,24 @@ type Batcher struct {
 
 	outSeq    int64
 	commitSeq int64
+
+	// for observability purposes
+	committedCountersMu sync.RWMutex
+	// committedCounters contains pair: timestamp - number of commits during timestamp
+	committedCounters map[int64]int64
+
+	// batcherTimeWaitMap contains two timestamps and durations of fastest and slowest committed batches during
+	// last 5 minutes. At index 0 lies fastest at index 1 lies slowest
+	batcherTimeWaitMap map[int64]BatcherTimeDTO
+	batcherTimeMu      sync.RWMutex
```

Comment:
it is good practice to write fields after mutex, which are locked by its.

Reply:
thx for advice.